model/src/cfd.rs

```diff
@@ -2350,6 +2379,25 @@ impl Dlc {
 
         Ok(Ok(Ok(signed_cet)))
     }
+
+    /// All the oracle event IDs associated with the DLC.
+    ///
+    /// This includes:
+    /// - the settlement event; and
+    /// - all the liquidation events that precede it.
```

Comment:
The main reason for keeping them together is that I don't want to make changes to the `Dlc` because we then have to think about how it affects the database.

When we generate the `Dlc` we, for a very long time, have been collecting the CETs into a `HashMap<BitMexPriceEventId, Vec<Cet>>`. As far as I know, we can rely on the latest `BitMexPriceEventId` (the key of the map) being the settlement event ID. And we already store the settlement event ID separately in the DLC anyway.

In conclusion: we could explicitly model this information in the DLC. But I don't want to think about breaking changes.